.github/workflows/npm-release.yml

```
@@ -0,0 +1,94 @@
+        description: "Add node release."
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+      app-path:
+        type: choice
+        description: Service-Name
+        required: true
+        options:
+          - refarch-frontend # todo: muss angepasst werden. eigenen Ordner verwenden
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)